liquibase-core/src/main/java/liquibase/change/AbstractSQLChange.java

```diff
@@ -162,15 +162,30 @@ public void setEndDelimiter(String endDelimiter) {
      */
     @Override
     public CheckSum generateCheckSum() {
+        InputStream stream = null;
```

Comment:
I am guessing that it will add a new blocker or critical SonarQube bug and as a result, change does not pass the quality gate.
I would recommend refactoring that somehow to use try-with-resources for input stream.

Reply:
@suryaaki2 , I updated the implementation. I ran a bunch of unit tests and they do not fail. Did you run bug scenario manually to be sure that it works well?

Reply:
@achehrynets Yes, I ran bug scenario manually after my initial changes and I did not the see the checksum validation error.